sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -1,7 +1,302 @@
+
+More information about Event Hubs features and terminology can be found here: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-features)
+
+# Examples
+
+Examples for various scenarios can be found on in the samples directory in our GitHub repo for
+[EventHubs](https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples).
+
+## Send events
+
+The following example shows how to send events to an event hub:
+
+```rust
+    let host = env::var("EVENTHUBS_HOST").unwrap();
+    let eventhub = env::var("EVENTHUB_NAME").unwrap();
+
+    let credential = DefaultAzureCredential::create(TokenCredentialOptions::default()).unwrap();
+
+    let client = ProducerClient::new(
+        host,
+        eventhub.clone(),
+        credential,
+        ProducerClientOptions::builder()
+            .with_application_id("test_create_batch")
+            .build(),
+    );
+    client.open().await.unwrap();
+    {
+        let mut batch = client.create_batch(None).await.unwrap();
+        assert_eq!(batch.len(), 0);
+        assert!(batch.try_add_event_data(vec![1, 2, 3, 4], None).unwrap());
+
+        let res = client.submit_batch(&batch).await;
+        assert!(res.is_ok());
+    }
+```
+
+## Receive events
+
+The following example shows how to receive events from partition 1 on an event hub:
+
+```rust no_run
+    let host = env::var("EVENTHUBS_HOST").unwrap();
+    let eventhub = env::var("EVENTHUB_NAME").unwrap();
+
+    info!("Establishing credentials.");
+
+    let credential = DefaultAzureCredential::create(TokenCredentialOptions::default()).unwrap();
+
+    info!("Creating client.");
+    let client = ConsumerClient::new(
+        host,
+        eventhub,
+        None,
+        credential,
+        Some(
+            ConsumerClientOptions::builder()
+                .with_application_id("receive_lots_of_events")
+                .build(),
+        ),
+    );
+
+    info!("Opening client.");
+    client.open().await.unwrap();
+
+    info!("Creating event receive stream.");
+    let event_stream = client
+        .receive_events_on_partition(
+            "0",
+            Some(
+                ReceiveOptions::builder()
+                    .with_start_position(StartPosition::builder().with_earliest_location().build())
+                    .build(),
+            ),
+        )
+        .await;
+
+    pin_mut!(event_stream); // Needed for iteration.
+    while let Some(event_result) = event_stream.next().await {
+        match event_result {
+            Ok(event) => {
+                // Process the received event
+                println!("Received event: {:?}", event);
+            }
+            Err(err) => {
+                // Handle the error
+                eprintln!("Error receiving event: {:?}", err);
+            }
+        }
+    }
+```
+
+# Troubleshooting
+
+## Logging
+
+The EventHubs SDK client uses the [tracing](https://docs.rs/tracing/latest/tracing/) package to
```

Comment:
Use "Event Hubs" or "EventHubs" consistently. It had a space more often above that I remember.

Reply:
Just checked - it's officially `Event Hubs`. I believe I've updated all of the instances.